src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { Tab } from '../types/Tabs';
+
+type Props = {
+  tabs: Tab[]
+  selectedTabId: string
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+  const handleTabClick = (tab: Tab) => {
+    return () => tab.id !== selectedTab?.id && onTabSelected(tab);
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+
+          {tabs.map(tab => (
+            <li
+              className={classNames({ 'is-active': tab.id === selectedTabId })}
```

Comment:
I think, your test is falied because you compare **tab.id === selectedTabId**
You should use **selectedTab.id** in this case)
I had the same problem.
```suggestion
              className={classNames({ 'is-active': tab.id === selectedTab.id })}
```

Reply:
thanks a lot.